test/commute.js

```diff
@@ -1,9 +1,16 @@
 var assert = require('assert');
-var Maybe = require('./shared/Maybe');
 
+var S = require('sanctuary');
+
+var Maybe = require('./shared/Maybe');
 var R = require('..');
 
 
+var eq = function(actual, expected) {
+  assert.strictEqual(R.toString(actual), R.toString(expected));
+};
```

Comment:
I certainly have no objection.

I wish I had started this project with different testing tools altogether.  But I've never had the heart to try to go back and retrofit them.  But this would certainly be a help.